client/src/components/Pages/recherche/SearchHeader/Filter/Filter.module.scss

```diff
@@ -39,11 +45,31 @@
       margin: 0 0 0 u(2);
     }
   }
+}
 
-  &.selected {
-    background: $bleuCharte !important;
-    color: $white !important;
-  }
+.checkbox {
+  flex: 1 0 0;
+  align-items: start;
+}
+
+.countContainer {
+  display: flex;
+  padding-top: 4px;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.count {
+  color: var(--tokens-light-text-mention-grey, #666);
```

Comment:
Did you use the CSS variable from Figma? I think these are not defined in the project. 
When I have to use one, I usually go on this page: https://components.react-dsfr.codegouv.studio/?path=/docs/%F0%9F%8E%A8-color-helper--page 
and search for the color with hex format. Then you have the right CSS variable